spot-client/src/spot-remote/ui/components/remote-control-menu/feedback-form.js

```diff
@@ -59,6 +85,19 @@ export default class FeedbackForm extends React.Component {
         );
     }
 
+    /**
+     * Cancels the inactivity timeout.
+     *
+     * @private
+     * @returns {void}
+     */
+    _cancelInactivityTimeout() {
+        if (this._timeout) {
```

Comment:
You don't need an truthy check for clearInterval/clearTimeout. The functions take in maybe anything and won't error.